modules/icon/css/lib/icon-list.scss

```diff
@@ -1,7 +1,10 @@
 .wdc-icon-list {
   display: flex;
-  margin-left: -$wdc-spacing-xxxs;
-  margin-right: -$wdc-spacing-xxxs;
+  margin-left: -$wdc-spacing-l;
```

Comment:
What did this get changed for?

Reply:
I was trying to match this style: https://github.com/Workday/canvas-kit/blob/master/modules/page-header/react/lib/PageHeader.tsx#L74

Reply:
@mannycarrera4 This also gets used in `Table` can you make sure it didn't break that use case? If this change is specific to the page header the margin should probably get added in that module.